syntax = "proto3";

message AuthRequest {
    string email = 1;
    string password = 2;
}

message AuthResponse {
    string access_token = 1;
}

service AuthService {
    rpc GetAccessToken(AuthRequest) returns (AuthResponse);
}
